### DISCUSSION TITLE
How to default value for checkbox ?

### DISCUSSION THREAD

**Discussion by altimusin** [filamentphp/filament#5260] [Help]
Hi All,

How to set default value for checkbox?

these statements is not working.

```
Checkbox::make('check1')->label('Option-1')->default(1)

Checkbox::make('check2')->label('Option-2')->default(true)
```

Thank you

**Comment by danharrin** [comment]
Both work for me. If this is a custom form, ensure you are calling `$this->form->fill()` from `mount()` as documented.
